### PR TITLE
Fix pw reset

### PR DESCRIPTION
**Pull request by BintLopez**
## Changelog
- Fixed nil error that caused pw reset page to crash

## Link to issue:  
https://github.com/MidwestAccessCoalition/lockbox_rails/issues/231

## Steps for QA/Special Notes:
- added tests!

## Relevant Screenshots: 
image

## Are you ready for review?:

- [x] Added relevant tests
- [x] Linked PR to the issue
- [x] Added notes for QA/special notes
- [x] Added revelant screenshots

### CODE REVIEW

File: app/controllers/users/passwords_controller.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class Users::PasswordsController < Devise::PasswordsController
-  layout "application"
+  layout "application", except: [:new]
```
Comment (BintLopez) [inline app/controllers/users/passwords_controller.rb]: this was the actual fix

File: app/views/layouts/_nav_bar.html.erb [hunk 2]
```diff
@@ -3,30 +3,34 @@
...
-    <button type="button" class="navbar-toggle collapsed flex-item" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
-      <span class="sr-only">Toggle navigation</span>
-      <% if current_user.admin? %>
-        <%= fa_icon "bars" %>
-        <%= fa_icon "times" %>
-      <% else %>
-        <%= fa_icon "cog" %>
-      <% end %>
-    </button>
+    <% if current_user %>
```
Comment (BintLopez) [inline app/views/layouts/_nav_bar.html.erb]: ensures that if there's not a `current_user`, we don't try to render the nav

File: app/views/layouts/_nav_bar.html.erb [hunk 3]
```diff
@@ -3,30 +3,34 @@
...
-      <%= link_to(edit_user_registration_path, class: 'btn btn-primary btn-invert') do %>
-        <span>Account Settings</span>
-      <% end %>
-      <% if current_user.admin? %>
-        <p>Quick Actions</p>
-        <p><a href="<%= lockbox_partners_path %>">View all lockboxes</a></p>
-        <p><a href="<%= support_requests_new_path %>">File a support request</a></p>
-        <p><a href="<%= new_lockbox_partner_path %>">Add a new lockbox partner</a></p>
-      <% end %>
+  <% if current_user %>
```
Comment (BintLopez) [inline app/views/layouts/_nav_bar.html.erb]: ensures that if there's not a `current_user`, we don't try to render the nav

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <header role="banner">
   <div id="main-header">
-    <%= render "layouts/nav_bar" %>
+      <%= render "layouts/nav_bar" %>
```
Comment (misternu) [inline app/views/layouts/_header.html.erb]: ?

**Comment by misternu** [comment]
hoot woot doot dee doot

**Pull request merged**